### DISCUSSION TITLE
Keycloak.X images ?

### DISCUSSION THREAD

**Discussion by bjolivot** [keycloak/keycloak#8704] [Q&A]
Hello!
I want to test Keycloak.x preview but I dont know where to start.
Are there docker images ? Some documentation ? K8S getting started ?

**Comment by stianst** [comment]
I'd suggest waiting a week or so until Keycloak 16 is out, as there's been a lot of changes and improvements recently. So stay tune!

**Reply by bjolivot** [reply]
ok thank's!

**Comment by bjolivot** [comment]
Can't find any news about Keycloak 16, would be great to have an update
